benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -108,6 +113,23 @@ public BenchmarkExecutionResult execute(Benchmark benchmark, int benchmarkOrdina
         }
     }
 
+    private BenchmarkExecutionResult warmupBenchmark(Benchmark benchmark, Optional<ZonedDateTime> executionTimeLimit)
+    {
+        BenchmarkExecutionResultBuilder resultBuilder = new BenchmarkExecutionResultBuilder(benchmark)
+                .withExecutions(List.of());
+        List<QueryExecutionResult> executions = executeQueries(benchmark, 1, true, executionTimeLimit);
```

Comment:
Why are you using `1` instead of `benchmark.getPrewarmRuns()`?

Reply:
You're right, it's surprising. I added `benchmark.getPrewarmRuns()` here.